lib/index.js

```diff
@@ -53,9 +53,11 @@ GA.prototype.screen = proxy('screen');
 
 function proxy(method){
   return function(message, fn){
-    if (this.settings.serversideClassic) {
-      return method === 'screen' ? setImmediate(fn) : this.classic[method].call(this.classic, message, fn);
+    var ctx = this.settings.serversideClassic ? this.classic : this.universal;
+    if ('function' === typeof ctx[method]) {
+      return ctx[method](message, fn);
+    } else {
+      return setImmediate(fn);
```

Comment:
Yup, I assume so at least.  I'm just covering all the bases because the one throwing all the errors is actually `identify` calls, so defiantly need to cover more than just screen.